ansible/roles/digidecs/templates/digidecs.service.j2

```diff
@@ -0,0 +1,16 @@
+# {{ ansible_managed }}
```

Comment:
I believe the brackets are redundant

Reply:
Perhaps, but we use with brackets in most places, so I prefer the consistency

Reply:
Damn you're right.
https://jpmens.net/2020/09/29/using-ansible-managed/